docs/index.md

```diff
@@ -38,6 +44,12 @@ The `Arcus.WebApi.Logging` package contains functionality that can be incorporat
 
 - [Logging unhandled exceptions](features/logging.md)
 
+## Arcus.WebApi.OpenApi.Extensions
```

Comment:
Let's go with the following to make it more readable
```suggestion
## OpenAPI Extensions
```

We should do the same for the other headers imo